modules/client-configuration/pages/automatic-client-registration.adoc

```diff
@@ -0,0 +1,72 @@
+[[automatic-client-registration]]
+= Automatic Registration of Clients Cretaed by Terraform
+
+
+New clients created by Terraform can be automatically registered to {productname}.
+
+{productname} supports two ways of registering new virtual machines created by Terraform:
```

Comment:
I would not say product support two ways, since it's not complitly correct. Product only provides the botstrap script.
I would say something like "This can be archived in two different ways..."

Reply:
Ok, thanks.